### PR TITLE
Finish all the tasks

### PR DESCRIPTION
**Pull request by dashayaskova**
I have encrypted passwords in database, but director still able to see them in his view. Also I have united two views with teachers and users into one with combobox with different positions. The next thing I have done is adding new user. Now it is possible to put marks after the date deadlines using this "super" user. 
Fixed some bugs, such as incomprehensible id of class in adding form and add method to some events.

### CODE REVIEW

**Comment by mike-kob** [comment]
> I have encrypted passwords in database, but director still able to see them in his view. Also I have united two views with teachers and users into one with combobox with different positions. The next thing I have done is adding new user. Now it is possible to put marks after the date deadlines using this "super" user.
> Fixed some bugs, such as incomprehensible id of class in adding form and add method to some events.

Wow, nice description)

File: Tools/Managers/StationManager.cs [hunk 1]
```diff
@@ -22,7 +22,6 @@ internal static class StationManager
         public static string CurrentYear { get; set; }
         public static IPerson CurrentMobile { get; set; }
         public static UsersViewModel usersView { get; set; }
```
Comment (mike-kob) [inline Tools/Managers/StationManager.cs:24]: Why do we need UserViewModel?

File: ViewModels/SignInViewModel.cs [hunk 2]
```diff
@@ -75,13 +77,25 @@ private bool CanExecuteCommand()
 
         private void SignInInplementation(object obj)
         {
+           
+            if(_login=="admin"&& StationManager.MainPassword.Password == "admin" 
+                && StationManager.DataStorage.DirectorExists())
```
Comment (mike-kob) [inline ViewModels/SignInViewModel.cs:82]: Wait a sec, if director doesn't exists, we should allow 'admin' access

File: ViewModels/TemplateViewModel.cs [hunk 3]
```diff
@@ -47,7 +47,7 @@ private void addItems()
             switch (StationManager.CurrentUser.AccessType)
             {
                 case "Директор":
-                    Items.Add("Учителя", ViewType.TeachersView);
+                 //   Items.Add("Учителя", ViewType.TeachersView);
```
Comment (mike-kob) [inline ViewModels/TemplateViewModel.cs:50]: I think. that we could remove this line

File: ViewModels/TemplateViewModel.cs [hunk 4]
```diff
@@ -68,7 +68,10 @@ private void addItems()
                     Items.Add("Успеваемость", ViewType.ProgressView);
                     // Items.Add("Отчет по ученикам");
                     Items.Add("Выставление оценок", ViewType.PutMarksView);
-                    
+                    break;
+                case "Суперпользователь":
```
Comment (mike-kob) [inline ViewModels/TemplateViewModel.cs:72]: Nive name, I like it)

File: ViewModels/UsersViewModels/ClassProgressViewModel.cs [hunk 5]
```diff
@@ -95,8 +95,12 @@ public ClassProgressViewModel()
...
                 IsAdmin = true;
             }
-            
-            
+
+            StationManager.RefreshYearListEvent += () =>
+            {
+                Years = StationManager.DataStorage.GetYears();
+                OnPropertyChanged("StYears");
+            };
```
Comment (mike-kob) [inline ViewModels/UsersViewModels/ClassProgressViewModel.cs:103]: oh, I like it)

**Review by mike-kob — APPROVED** [review]
Everything seems nice

File: ViewModels/SignInViewModel.cs (hunk 2 above)
Comment (dashayaskova) [reply ViewModels/SignInViewModel.cs:82]: well, sorry, wrong name of method

File: ViewModels/UsersViewModels/ClassProgressViewModel.cs (hunk 5 above)
Comment (dashayaskova) [reply ViewModels/UsersViewModels/ClassProgressViewModel.cs:103]: ?

**Pull request merged**